collector/src/main/java/com/bitmovin/analytics/BitmovinAnalyticsConfig.java

```diff
@@ -286,10 +298,27 @@ public void setTitle(String title) {
     }
 
     /**
-     * A/B Test Experiment Name
+     * MPD URL of the manifest. Usually auto-detected, but can be overwritten.
```

Comment:
i'd change the comment to say "If not set explicitly the collector will retrieve available information from the player"

Reply:
Changed in 8e781300a0c05a4b7dd3ac471563976ee9db58e8